common/lib/xmodule/xmodule/js/src/word_cloud/word_cloud_main.js

```diff
@@ -230,15 +233,30 @@ define('WordCloudMain', [], function () {
         $.each(response.student_words, function (word, stat) {
             var percent = (response.display_student_percents) ? ' ' + (Math.round(100 * (stat / response.total_count))) + '%' : '';
 
-            studentWordsKeys.push('<strong>' + word + '</strong>' + percent);
+            studentWordsKeys.push(HtmlUtils.interpolateHtml(
+                gettext('{startTag}{word}{endTag}{percent}'),
+                {
+                    startTag: HtmlUtils.HTML('<strong>'),
+                    word: word,
+                    endTag: HtmlUtils.HTML('</strong>'),
+                    percent: percent
+                }
+            ));
         });
         studentWordsStr = '' + studentWordsKeys.join(', ');
 
         cloudSectionEl
-            .addClass('active')
-            .find('.your_words').html(studentWordsStr)
-            .end()
-            .find('.total_num_words').html(response.total_count);
+            .addClass('active');
+
+        HtmlUtils.setHtml(
+            cloudSectionEl.find('.your_words'),
+            HtmlUtils.HTML(studentWordsStr)
+        );
+
+        HtmlUtils.setHtml(
+            cloudSectionEl.find('.your_words').end().find('.total_num_words'),
+            HtmlUtils.HTML(response.total_count)
```

Comment:
This isn't actually HTML, it's just a number. I think it should actually be escaped.

Instead of using setHtml, just use .text (it is a span element).